spylib/utils/rest.py

```diff
@@ -0,0 +1,41 @@
+from enum import Enum
+
+from pydantic import BaseModel
+
+
+class Status(Enum):
```

Comment:
Hummmmm How about reusing [FastAPI's status](https://fastapi.tiangolo.com/tutorial/response-status-code/?h=status#shortcut-to-remember-the-names) which are actually just importing the [Starlette ones](https://github.com/encode/starlette/blob/master/starlette/status.py). Let's not reinvent the wheel. We already committed to fastapi in the OAuth Router anyway.